testing.md

```diff
@@ -61,5 +171,55 @@ The following query is sufficient to test Native Query as it exercises mutliple
 | laptop | 3257.093 |
 | envelopes | 77.661 |
 
+# Manual Testing (for Direct Query)
+
+At this time, Direct Query must be tested semi-manually by reviewers, until such time as the Power Query
+SDK uses the same Mashup Engine as Power BI.
 
+Each Direct Query test we want to run is stored in the `resources/direct_query` directory
+
+1. Ensure you have a local ADF running using the instructions from the [Setup](Setup) Section.
+- Load data
+    ```
+    ./mongoimport.exe --uri="mongodb://localhost:28017/reports" --drop resources/integration_test/testdata/transforms.json
```

Comment:
in theory we could also use `$ADF_TEST_LOCAL_HOST:$ADF_TEST_LOCAL_PORT` here too right?

Reply:
sgtm